web/pom.xml

```diff
@@ -306,26 +306,48 @@
          </plugin>
 
          <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>exec-maven-plugin</artifactId>
-            <version>1.3.2</version>
+            <groupId>com.github.eirslett</groupId>
+            <artifactId>frontend-maven-plugin</artifactId>
```

Comment:
It downloads and installs Node, NPM locally for this project. It runs Bower and executes Gulp build task.